decidim-core/app/views/layouts/decidim/header/_menu_breadcrumb_main_dropdown.html.erb

```diff
@@ -1,9 +1,10 @@
 <div id="<%= id %>" class="menu-bar__main-dropdown">
   <div class="menu-bar__main-dropdown__top">
-    <div class="menu-bar__main-dropdown__top-left">
-      <%= render partial: "layouts/decidim/header/menu_breadcrumb_main_dropdown_top_left" %>
```

Comment:
by removing this, the view `menu_breadcrumb_main_dropdown_top_left` is not used anymore, we can delete it then

Reply:
done